src/aui-form-builder/js/aui-form-builder-field-types.js

```diff
@@ -6,7 +6,7 @@
  */
 
 var CSS_FIELD_TYPE = A.getClassName('field', 'type'),
-    CSS_FIELD_TYPES_LABEL = A.getClassName('form', 'builder', 'field', 'types', 'label'),
+    CSS_FIELD_TYPES_LABEL = A.getClassName('modal', 'title'),
```

Comment:
Could you update this var name with something that best describes the css class?